src/Slack/ApiHelper.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Secret Santa project.
+ *
+ * (c) JoliCode <coucou@jolicode.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace JoliCode\SecretSanta\Slack;
+
+use JoliCode\Slack\Api\Client;
+use JoliCode\Slack\ClientFactory;
+
+class ApiHelper
+{
+    private $clientsByToken = [];
+
+    public function getClientForToken(string $token): Client
```

Comment:
I would move this method to the `ClientFactory` class

Reply:
Why not, but the ClientFactory class belongs to another package (`jolicode/slack-php-api`)

Reply:
Ah :/ I would rename it to `ClientFactory` at least